### ISSUE TITLE
Option to add/view stack from first frame instead of center

### ISSUE THREAD

**Issue by hboehmer868** [napari/napari#5070]
## 🐛 Bug

When I load a sequence of TIF images according to [this](https://napari.org/stable/tutorials/processing/dask.html#make-your-life-easier-with-dask-image) method, the dimension slider is set to the 3rd position and the according image is displayed. To start on the first image of the sequence I manually set the slider position with `viewer.dims.set_current_step(0,0)`. However this causes BOTH images to load and such reduces the application boot time.

## To Reproduce

1. Load image sequence according to mentioned method
2. Run napari Viewer

## Expected behavior

The first image of the sequence should be shown and subsequently only that should be loaded.

## Environment

 - Please copy and paste the information at napari info option in help menubar here:

napari: 0.4.16
Platform: Windows-10-10.0.19044-SP0
Python: 3.10.5 (tags/v3.10.5:f377153, Jun 6 2022, 16:14:13) [MSC v.1929 64 bit (AMD64)]
Qt: 5.15.2
PyQt5: 5.15.7
NumPy: 1.22.4
SciPy: 1.8.1
Dask: 2022.7.0
VisPy: 0.10.0

OpenGL:
- GL version: 4.6.0 NVIDIA 516.01
- MAX_TEXTURE_SIZE: 32768

Screens:
- screen 1: resolution 1920x1080, scale 1.0
- screen 2: resolution 1920x1080, scale 1.0

Plugins:
- console: 0.0.4
- napari-svg: 0.1.6
- scikit-image: 0.4.16

 - Any other relevant information:

None

## Additional context

None

**Comment by psobolewskiPhD** [comment]
This fell through the cracks, major apologies!
However, this is not a bug: napari defaults to showing the middle slice to a stack, see:
https://github.com/napari/napari/blob/5e677ec7fe40dd54eeda68c1e1c6b6e70a6d60b9/napari/components/viewer_model.py#L638

I guess I can see the use-case of wanting to over-ride this behavior when adding a layer, to start at the initial frame (or arbitrary frame) instead...

**Comment by psobolewskiPhD** [comment]
So I happened to run into this again with a time series.
So I've got something working:
https://github.com/psobolewskiPhD/napari/tree/keyword_to_start_from_zero
a kwarg to add_image/view_image `sequence` that if true, doesn't move to center slice.

Kind of a proof of concept, because I'm not sure the implementation will pass mustard with other devs 😢 
cc: @jni

**Comment by jni** [comment]
Yeah, a kwarg to add_image is not the right approach here imho, because it is unrelated to image data (and we are trying to reduce the size of that parameter list). Perhaps a setting?

**Comment by psobolewskiPhD** [comment]
> Yeah, a kwarg to add_image is not the right approach here imho, because it is unrelated to image data (and we are trying to reduce the size of that parameter list).

I figured. It's not related to the image data per se, but sort of is, because it's related to the axes. 
I sort of think this is a lot like RGB or channel_axis 👀 

I can't see how a setting would help, after all not all images are time series/sequences. Current move to center is just useful magic that tends to be good for z-stacks.
